src/HasteModuleLoader/HasteModuleLoader.js

```diff
@@ -204,10 +204,8 @@ class Loader {
 
     // Use this name for the module wrapper for consistency with node.
     const evalResultVariable = 'Object.<anonymous>';
```

Comment:
thanks, would you mind adding one test that throws during init and one in a call afterwards?